modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -62,7 +61,7 @@ public void start() {
         String networkAlias = getNetworkAliases().get(0);
         proxy = new SocatContainer()
                 .withNetwork(getNetwork())
-                .withTarget(9092, networkAlias)
+                .withTarget(9093, networkAlias)
```

Comment:
I think it would be more consistent to use `KAFKA_PORT` here (as well as in line 35).

Reply:
@kiview changed to constants, could you please re-check?